packages/server/migrations/20221012195737_update_interested_codes_is_rejection_column.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = async function (knex) {
+    // Create a new column as grant statuses are no longer binary and a boolean will no longer be sufficient
+    await knex.schema.table('interested_codes', (table) => { table.string('status_code'); });
+
+    // Copying existing is_rejection column values over to the new column
+    await knex('interested_codes').where({ is_rejection: false }).update({ status_code: 'Interested' });
+    await knex('interested_codes').where({ is_rejection: true }).update({ status_code: 'Rejected' });
+
+    // Add new grant statuses and set the auto_increment counter for id to the accurate number as
+    // knex was trying to add new entries at id=1
+    await knex.raw('SELECT setval(\'interested_codes_id_seq\', max(id)) FROM interested_codes');
```

Comment:
Hmm, it may be a local issue with my database, but I pushed ahead because I couldn't think of any significant downsides to having the staging / production DB also resync the primary ID count... what do you think?

Reply:
The only downside I see happens if we frequently delete data from `interest_codes` table. Does that happen often? If that doesn't happen often, then I'm good with keeping things as is 👍 

The super edge case scenario I'm thinking of will only happen if there were rows with `id > max(id)` and were deleted prior to this statement running. Here's how I think it may play-out:
```
scenario where interested_codes_id_seq == 2
-----------
|  id   |  code |
-----------
|  1    |  'a'  |
-----------
|  2    |  'b'  |
-----------

record with id 2 is deleted but interested_codes_id_seq is still == 2
-----------
|  id   |  code |
-----------
|  1    |  'a'  |
-----------

statement above would reset interested_codes_id_sq == 1

This shouldn't be a problem most of the time except in the scenario where there were   references to the old record `id==2` that were not appropriately nulled/deleted from other tables.
```